js-jasmine/src/concert_tickets.js

```diff
@@ -0,0 +1,23 @@
+class Tickets {
+    constructor(sellIn, quality) {
+      this.sellIn = sellIn;
+      this.quality = quality;
+    }
+  
+    updateQuality() {
+      if (this.sellIn === 0) {
```

Comment:
maybe you could account for when sellIn is negative as well because i think at the moment it would set quality to 50?